src/main/java/org/opensearch/performanceanalyzer/collectors/ScheduledMetricCollectorsExecutor.java

```diff
@@ -120,8 +122,8 @@ public void run() {
                         PerformanceAnalyzerMetricsCollector collector = entry.getKey();
                         if (collector.getState()
                                 == PerformanceAnalyzerMetricsCollector.State.MUTED) {
-                            StatsCollector.instance()
-                                    .logMetric(StatExceptionCode.COLLECTORS_MUTED.toString());
+                            PerformanceAnalyzerApp.WRITER_METRICS_AGGREGATOR.updateStat(
```

Comment:
Is there a way to include which collector was muted?

Reply:
Good point, converted to `NAMED_COUNTERS`